### DISCUSSION TITLE
Getting PermissionError When Trying To Add A Script

### DISCUSSION THREAD

**Discussion by theMachineMasse** [netbox-community/netbox-docker#1093] [Getting Help]
I recently updated my netbox-docker to 2.6.1 with NetBox running on 3.5.9. The instance came up without issue, but when trying to add scripts via the UI, I receive this error. 
![image](https://github.com/netbox-community/netbox-docker/assets/55885611/72935257-3289-4b8c-b92d-5644c1f42349)
I am running the containers on Docker Desktop on Windows 10. Any help would be greatly appreciated!

**Comment by barmalej1** [comment]
I was getting the same error when attempting to upload rack images. I fixed it with the following command:

chmod -R g+w /var/lib/docker/volumes/netboxdocker_netbox-media-files/_data
